operations/observability/mixins/workspace/rules/central/ipfs.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+# Licensed under the GNU Affero General Public License (AGPL).
+# See License.AGPL.txt in the project root for license information.
+
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
+  name: ipfs-monitoring-rules
+spec:
+  groups:
+    - name: ipfs-alerts
+      rules:
+        - alert: IPFSAlmostOutOfStorage
+          labels:
+            severity: warning
```

Comment:
Warnings tend to be overlooked. We should add another (critical) alert when we fall below 200 GB.

Reply:
good point

Reply:
added